packages/seo-integration/tests/replacement-variables/configurations.test.js

```diff
@@ -0,0 +1,42 @@
+import * as data from "@wordpress/data";
+import { excerpt } from "../../src/replacement-variables/configurations";
+
+const content = jest.fn().mockReturnValue( "A calico cat is a domestic cat of any breed with a tri-color coat. The calico cat is most commonly thought " +
```

Comment:
Minor point: I would call this `selectContent`, since it is a (mocked) function to select/retrieve content and not the content in itself 🙂 .

Reply:
I will make the change! 🙂